src/Datadog.Trace/Span.cs

```diff
@@ -62,37 +48,29 @@ internal Span(IDatadogTracer tracer, SpanContext parent, string operationName, s
         /// <summary>
         /// Gets or sets the service name
         /// </summary>
-        public string ServiceName
-        {
-            get => _context.ServiceName;
-            set => _context.ServiceName = value;
-        }
+        public string ServiceName { get; set; }
 
         /// <summary>
         /// Gets the trace's unique identifier.
         /// </summary>
-        public ulong TraceId => _context.TraceId;
+        [Obsolete("This property will be removed in future versions of this library. Use Span.Context.TraceId instead.")]
```

Comment:
`Context` should become public then, I suppose.

Reply:
Yeah, never mind. I removed this attribute.